From ef3e66e168c4b9b86bfc4903001631e53a7125d8 Mon Sep 17 00:00:00 2001
From: Jussi Kivilinna <jussi.kivilinna@iki.fi>
Date: Sun, 12 Jan 2014 22:01:28 +0200
Subject: [PATCH] Fix assembly division check

* configure.ac (gcry_cv_gcc_as_const_division_ok): Correct variable
name mismatch at '--Wa,--divide' workaround check.
--

Signed-off-by: Jussi Kivilinna <jussi.kivilinna@iki.fi>
---
 configure.ac | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/configure.ac b/configure.ac
index fac5f7a..7d37f94 100644
--- a/configure.ac
+++ b/configure.ac
@@ -1101,7 +1101,7 @@ if test $amd64_as_feature_detection = yes; then
           AC_COMPILE_IFELSE([AC_LANG_SOURCE(
             [[__asm__("xorl \$(123456789/12345678), %ebp;\n\t");]])],
             [gcry_cv_gcc_as_const_division_with_wadivide_ok=yes])])
-    if test "$gcry_cv_gcc_as_const_division_ok_with_wadivide_ok" = "no" ; then
+    if test "$gcry_cv_gcc_as_const_division_with_wadivide_ok" = "no" ; then
       # '-Wa,--divide' did not work, restore old flags.
       CPPFLAGS="$_gcc_cppflags_save"
     fi
-- 
1.9.1

